Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/03-AAD-create/variables.tf

```diff
@@ -30,4 +30,6 @@ variable "storage_account_name" {
 
 variable "container_name" {
   default = "akscs"
-}
\ No newline at end of file
+}
+
+
```

Comment:
Any need for this change?

Reply:
some blank lines - removed.

Reply:
But you didn't remove the text that said you're going to restart the jumpbox.

> Restart your domain controller first and then the jumpbox VM to propagate the new DNS settings. Default values for the network resource group and VM names used in example below. 

Please read through the instructions and make sure they are updated to reflect the new changes